sqlitecluster/SQLiteNode.cpp

```diff
@@ -2212,6 +2213,13 @@ void SQLiteNode::_updateSyncPeer()
             continue;
         }
 
+        // We want to sync, if possible, from a peer that is not the leader. So at this point, skip choosing it
+        // as the newSyncPeer.
+        if (peer == _leadPeer) {
+            isLeaderValidPeer = true;
```

Comment:
I don't think we need this (see below), we can just `continue` without setting `isLeaderValidPeer`.

Reply:
Done